astropy/table/tests/test_pprint.py

```diff
@@ -910,3 +910,18 @@ def test_output_globs(self):
         with t.pprint_exclude_names.set('a??'):
             out = t.pformat_all()
         assert out == exp
+
+
+def test_multidims_with_zero_dim():
```

Comment:
Hmm. This test fails on this branch. Either something else also needs backporting or this patch should not be in LTS. @taldcroft , what do you think?

```
_________________________ test_multidims_with_zero_dim _________________________

    def test_multidims_with_zero_dim():
        """Test of fix for #13836 when a zero-dim column is present"""
        t = Table()
        t["a"] = ["a", "b"]
        t["b"] = np.ones(shape=(2, 0, 1), dtype=np.float64)
        exp = [
            " a        b      ",
            "str1 float64[0,1]",
            "---- ------------",
            "   a             ",
            "   b             ",
        ]
>       assert t.pformat_all(show_dtype=True) == exp
E       AssertionError: assert [' a   b [0,1...   b        '] == [' a        b...            ']
E         At index 0 diff: ' a   b [0,1]' != ' a        b      '
E         Full diff:
E           [
E         -  ' a        b      ',
E         +  ' a   b [0,1]',
E         -  'str1 float64[0,1]',
E         ?               -----
E         +  'str1 float64',
E         -  '---- ------------',
E         ?        -----
E         +  '---- -------',
E         -  '   a             ',
E         ?       -----
E         +  '   a        ',
E         -  '   b             ',
E         ?       -----
E         +  '   b        ',
E           ]

.../astropy/table/tests/test_pprint.py:927: AssertionError
```